### ISSUE TITLE
Handling headers in HTTP requests

### ISSUE THREAD

**Issue by aravindkk** [NickCarneiro/curlconverter#3]
So when we have headers in the HTTP request, the code at http://curl.trillworks.com/#node is unable to handle it. Is it a feature that is yet to be implemented or have I just caught a bug? Want a helping hand?

**Comment by NickCarneiro** [comment]
The original use cases were pretty narrow, but it would be great to make it work for more.

Go ahead and send a pull request that includes a test case in test/test.js.

**Comment by rodw** [comment]
Actually, it seems like the headers do work, but for some reason you must have at least two of them.

That is, this:

```bash
curl http://example.com/ -H 'foo: bar' 
```

fails with an "Error parsing curl command." message, but this:

```bash
curl http://example.com/ -H 'foo: bar'  -H 'foo2: bar2'
```

works properly.

@aravindkk this may be a work around for you at least.  Also I imagine this isn't too hard to track down in the code, seems like a silly little bug that's easily fixed.

**Comment by rodw** [comment]
Actually, here's a fix for this issue.

In `util.js` the code assumes that `parsedArguments.H` is an array, which is true when there is more than one `-H` value passed on the curl command line, but not when there is exactly one.

One easy way to address that is just to wrap the value of `parsedArguments.H` in an array if it isn't already an array, adding these three lines:

```diff
--- a/util.js
+++ b/util.js
@@ -22,6 +22,9 @@ var parseCurlCommand = function(curlCommand) {
     var headers;
     if (parsedArguments.H) {
         headers = {};
+        if (!Array.isArray(parsedArguments.H)) {
+          parsedArguments.H = [parsedArguments.H];
+        }
         parsedArguments.H.forEach(function (header) {
             if (header.indexOf('Cookie') !== -1) {
                 cookieString = header;
```

If you are uncomfortable modifying the content of `parsedArguments` (since it comes from `minimist` originally) you could obviously introduce a new variable to iterate over.

If I get around to it I can probably create a proper pull request for this, but it's pretty much a one line fix.

**Comment by NickCarneiro** [comment]
@rodw, thanks for digging into that. I've implemented your fix in 0.0.19. I'll publish to the website shortly.
